### PR TITLE
added colspan to paragraph td to prevent error in Outlook, and added …

### PR DESCRIPTION
**Pull request by tifsev**
…mktNoTrack to Contact Us links

### CODE REVIEW

**Review by nambuben — COMMENTED** [review]
@Gonzo427 Wouldn't this only load the webfont on Outlook clients? How/where is it loaded for non Outlook clients?

File: UW Templates/Be The First (MASTER Event).html [hunk 1]
```diff
@@ -21,7 +21,9 @@
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">        
-        <link href="http://fonts.googleapis.com/css?family=Open+Sans:400,600,800" rel="stylesheet" type="text/css">
+        <!--[if !mso]><!-->
+      <link href="http://fonts.googleapis.com/css?family=Open+Sans:400,600,800" rel="stylesheet" type="text/css" />
```
Comment (nambuben) [inline UW Templates/Be The First (MASTER Event).html:25]: @Gonzo427 Are we now entirely moving to system sans-serif fonts on Outlook clients?

Comment (tifsev) [reply UW Templates/Be The First (MASTER Event).html:25]: No, this loads on all clients except Outlook. (Notice the ! in front of mso). If an Outlook user doesn't have Open Sans, then default will be sans-serif. Without that conditional, for some reason, the default becomes Times New Roman.

Comment (nambuben) [reply UW Templates/Be The First (MASTER Event).html:25]: Totally, I see that.  But if the font import is now only happening on non-outlook clients, are we officially saying that everything on Outlook should just be times new roman/helvetica?  We aren't even trying to load Open Sans in Outlook?

Still approving, just want to get clarification on how hard we're pushing Open Sans generally.

**Review by nambuben — APPROVED** [review]
Good PR, confirming orientation on Open Sans generally now that we're not loading it in Outlook.

**Comment by tifsev** [comment]
The templates still call for the font-family to be Open Sans in Outlook (in
conditional styles block). The recipient must have the font loaded on their
machine already, since Outlook won't load a google font. If no Open Sans,
it should be Arial. Make sense?

Thanks for checking!
Tiffany

**Comment by nambuben** [comment]
Gotcha.  Totally makes sense.  I'll be excited to see it in action.

**Pull request merged**